docs/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.16.3)
+project(docs)
```

Comment:
Maybe clio_docs?

Reply:
I like the short `docs` better - it's obvious it's for Clio as it's part of our repo and folder structure